Foundation/NSLocale.swift

```diff
@@ -55,7 +55,7 @@ public class NSLocale : NSObject, NSCopying, NSSecureCoding {
         return copyWithZone(nil)
     }
     
-    public func copyWithZone(_ zone: NSZone) -> AnyObject { NSUnimplemented() }
+    public func copyWithZone(_ zone: NSZone) -> AnyObject { return self }
```

Comment:
Yah, I think that's probably a mistake in NSLocale. But anyway, we can proceed with this for now.